src/lesson-1-9-demo/ProductCatalog.jsx

```diff
@@ -42,14 +43,17 @@ const ProductCatalog = () => {
         setProducts(response.data);
         setNextCursor(response.nextCursor);
         setPrevCursor(response.prevCursor);
-      })
-      .catch((error) => {
+      } catch (error) {
```

Comment:
here we replace calls to `.catch` with the `catch` portion of a `try / catch` block